roofit/roofitcore/src/RooPlot.cxx

```diff
@@ -266,7 +266,45 @@ RooPlot::RooPlot(const RooAbsRealLValue &var, Double_t xmin, Double_t xmax, Int_
   _normBinWidth = (xmax-xmin)/nbins ;
 }
 
+////////////////////////////////////////////////////////////////////////////////
+//
+
+RooPlot* RooPlot::frame(const RooAbsRealLValue &var, Double_t xmin, Double_t xmax, Int_t nBins){
+  return new RooPlot(var,xmin,xmax,nBins);
+}
+RooPlot* RooPlot::frameWithLabels(const RooAbsRealLValue &var){
```

Comment:
I would also advise to add doxygen comments, unless it's private/protected. Even if it is, it's nice to have them.

Reply:
Will be done.